api/docs.md

```diff
@@ -1046,6 +1047,60 @@ Deletes multiple participants from an organization by their participant IDs. Req
 | `400` | `40011` | `no organization provided` |
 | `500` | `50002` | `internal server error` |
 
+### 🎫 Create Organization Ticket
+
+* **Path** `/organizations/{address}/tickets`
+* **Method** `POST`
+* **Headers**
+  * `Authentication: Bearer <user_token>`
+* **Request body**
+```json
+{
+  "type": "support",
+  "title": "Need help with voting process",
+  "description": "I'm having trouble setting up a new voting process. Can you help?"
+}
+```
+
+* **Description**
+Creates a new support ticket for the organization. The user must be a member of the organization with any role. The ticket is sent to the support team via email.
+
+* **Errors**
+
+| HTTP Status | Error code | Message |
+|:---:|:---:|:---|
+| `401` | `40001` | `user not authorized` |
+| `400` | `40004` | `malformed JSON body` |
+| `400` | `40011` | `no organization provided` |
+| `500` | `50002` | `internal server error` |
+
+### 🎫 Create Organization Ticket
+
+* **Path** `/organizations/{address}/tickets`
+* **Method** `POST`
+* **Headers**
+  * `Authentication: Bearer <user_token>`
+* **Request body**
+```json
+{
+  "type": "support",
+  "title": "Need help with voting process",
+  "description": "I'm having trouble setting up a new voting process. Can you help?"
+}
+```
+
+* **Description**
+Creates a new support ticket for the organization. The user must be a member of the organization with any role. The ticket is sent to the support team via email.
+
+* **Errors**
+
+| HTTP Status | Error code | Message |
+|:---:|:---:|:---|
+| `401` | `40001` | `user not authorized` |
+| `400` | `40004` | `malformed JSON body` |
+| `400` | `40011` | `no organization provided` |
+| `500` | `50002` | `internal server error` |
+
```

Comment:
why twice? isn't this is exactly the same as lines 1050-1076?

Reply:
Oops. This was an error introduced when rebasing with main.